extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -458,6 +488,49 @@ export class AmpConsent extends AMP.BaseElement {
     this.hide_();
   }
 
+  /**
+   * Maybe set the consent purpose map with default values.
+   * If not, resolve instantly.
+   * @param {string} action
+   * @param {../../../src/service/action-impl.ActionInvocation=} opt_invocation
+   * @return {!Promise}
+   */
+  maybeSetConsentPurposeDefaults_(action, opt_invocation) {
+    if (
+      !this.isGranularConsentExperimentOn_ ||
+      !opt_invocation ||
+      !opt_invocation['args'] ||
+      !opt_invocation['args']['purposeConsentDefault']
```

Comment:
Any chance we can use optional chaining?

Reply:
Looks like we are permitted to use optional chaining, although it's not widely used, yet :)